mev_inspect/punks.py

```diff
@@ -0,0 +1,104 @@
+from typing import List
+from mev_inspect.schemas.traces import (
+    ClassifiedTrace,
+    Classification,
+    DecodedCallTrace,
+)
+from mev_inspect.schemas.punk_bid import PunkBid
+from mev_inspect.schemas.punk_accept_bid import PunkBidAcceptance
+from mev_inspect.schemas.punk_snipe import PunkSnipe
+from mev_inspect.traces import get_traces_by_transaction_hash
+
+
+def get_punk_snipes(
+    punk_bids: List[PunkBid], punk_bid_acceptances: List[PunkBidAcceptance]
+) -> List[PunkSnipe]:
+    punk_snipe_list = []
+
+    for punk_bid in punk_bids:
+        for punk_bid_acceptance in punk_bid_acceptances:
+            if punk_bid.punk_index == punk_bid_acceptance.punk_index:
+                if punk_bid.amount > punk_bid_acceptance.min_price:
```

Comment:
if there's only one bid and it's above the min, that would still generate a "swipe" right?

also we mentioned this creates multiple swipes if there are multiple people who swiped each other (out bid +1 wei then out bid +2 wei, etc)

I think it'd be nicer to have a single object per instance of potentially many bids (`PunkAuction`?)

something like:
```
for punk_bid_acceptance in punk_bid_acceptances:
    punk_bids = get_bids_for_acceptance(...)
    punk_auction = PunkAuction(
        total_bids=len(punk_bids)
        winning_bid_amount=max(bid.amount for bid in punk_bids),
        ...,
    )
```